frontend/src/tests/unit/validators.test.js

```diff
@@ -100,55 +112,28 @@ describe('Email validator tests:', () => {
     expect(validatorMessage1).toBe(FIELD_REQUIRED)
     expect(validatorMessage2).toBe(FIELD_REQUIRED)
   })
-  it('should return INCORRECT_EMAIL error for bad email string', () => {
-    // given
-    const exampleEmail1 = 'testEmail'
-    const exampleEmail2 = 'testEmail@'
-    const exampleEmail3 = 'testEmail@test.'
-    const exampleEmail4 = 'test@test.com'
-    const exampleEmail5 = 'test@test.pl'
-    const exampleEmail6 = 'test@agh.edu.pl.test.pl'
-    const exampleEmail7 = '@agh.edu.pl'
-    const exampleEmail8 = '123456@agh.edu.pl'
-
+  it.each(incorrectEmails)('should return INCORRECT_EMAIL error for %s', (email) => {
     // when
-    const validatorMessage1 = validateEmail(exampleEmail1, AccountType.STUDENT)
-    const validatorMessage2 = validateEmail(exampleEmail2, AccountType.PROFESSOR)
-    const validatorMessage3 = validateEmail(exampleEmail3, AccountType.PROFESSOR)
-    const validatorMessage4 = validateEmail(exampleEmail4, AccountType.STUDENT)
-    const validatorMessage5 = validateEmail(exampleEmail5, AccountType.PROFESSOR)
-    const validatorMessage6 = validateEmail(exampleEmail6, AccountType.STUDENT)
-    const validatorMessage7 = validateEmail(exampleEmail7, AccountType.PROFESSOR)
-    const validatorMessage8 = validateEmail(exampleEmail8, AccountType.STUDENT)
+    const validatorMessage1 = validateEmail(email, AccountType.STUDENT)
+    const validatorMessage2 = validateEmail(email, AccountType.STUDENT)
 
     // then
     expect(validatorMessage1).toBe(INCORRECT_EMAIL)
     expect(validatorMessage2).toBe(INCORRECT_EMAIL)
-    expect(validatorMessage3).toBe(INCORRECT_EMAIL)
-    expect(validatorMessage4).toBe(INCORRECT_EMAIL)
-    expect(validatorMessage5).toBe(INCORRECT_EMAIL)
-    expect(validatorMessage6).toBe(INCORRECT_EMAIL)
-    expect(validatorMessage7).toBe(INCORRECT_EMAIL)
-    expect(validatorMessage8).toBe(INCORRECT_EMAIL)
   })
-  it('should validate email string correctly for agh domain emails', () => {
-    // given
-    const exampleEmail1 = 'student@student.agh.edu.pl'
-    const exampleEmail2 = 'prof@agh.edu.pl'
-    const exampleEmail3 = 'z1@agh.edu.pl'
-    const exampleEmail4 = 'F2@student.agh.edu.pl'
+  it.each(correctStudentEmails)('should validate student email %s correctly', (email) => {
+    // when
+    const validatorMessage = validateEmail(email, AccountType.STUDENT)
 
+    // then
+    expect(validatorMessage).toBe('')
+  })
+  it.each(correctProfessorEmails)('should validate student email %s correctly', (email) => {
```

Comment:
Treść wiadomości nie pokrywa się z case'm:
```suggestion
  it.each(correctProfessorEmails)('should validate professor email %s correctly', (email) => {
```

Reply:
racja, nie wiem czemu to przegapiłem :/